examples/react/full-width-resizable-table/yarn.lock

```diff
@@ -0,0 +1,323 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
+"@faker-js/faker@6.3.1":
```

Comment:
don't add yarn.lock. none of the other examples added yarn.lock.

Reply:
yarn.lock has been removed from the PR